Source/DafnyCore/Compilers/Java/JavaBackend.cs

```diff
@@ -81,19 +78,9 @@ public class JavaBackend : ExecutableBackend {
       return false;
     }
 
-    if (!DafnyOptions.O.UseRuntimeLib) {
```

Comment:
It looks like you are changing the Java build behavior to no longer include the runtime files in the built executable jar, but rather, requiring DafnyRuntime.jar on the CLASSPATH. If so (1) that ought to be a user choice (based on --include-runtime, and the default should be to include the runtime in the executable jar) and (2) the trend in Java is to package up everything in an executable Jar, or even in a natively executable exe by bundling in the JVM, so I think unbundling is the wrong direction. And this is definitely a change in behavior, as we document that one can just run the executable jar (without having to adjust the CLASSPATH).

If instead, you have included the sources and the user is recompiling the sources to include in the executable jar. then we have just added to the user's compilation burden (a backward step).

[ I though t we were just fixing the omission of not emitting runtime sources in 'dafny translate']

Reply:
> It looks like you are changing the Java build behavior to no longer include the runtime files in the built executable jar

They are included through the changes in `Source/DafnyCore/Compilers/Java/Compiler-java.cs`. This change makes sure they're not included twice.

Reply:
I see, thanks.